### PR TITLE
serve LoadUserPlusKeys from cache if possible

### PR DESCRIPTION
**Pull request by maxtaco**
- also provide a debugging mechanism to hunt down where loadUser calls are coming from

### CODE REVIEW

**Comment by maxtaco** [comment]
cc: @mmaxim

**Comment by mmaxim** [comment]
👍

File: go/libkb/cached_user_loader.go [hunk 1]
```diff
@@ -172,3 +172,24 @@ func (u *CachedUserLoader) CheckKIDForUID(uid keybase1.UID, kid keybase1.KID) (f
 	found, revokedAt = CheckKID(upk, kid)
 	return found, revokedAt, nil
 }
+
+func (u *CachedUserLoader) LoadUserPlusKeys(uid keybase1.UID) (keybase1.UserPlusKeys, error) {
+	var up keybase1.UserPlusKeys
+	if uid.IsNil() {
+		return up, fmt.Errorf("Nil UID")
```
Comment (patrickxb) [inline go/libkb/cached_user_loader.go]: i see this was moved from somewhere else...fine to leave.

**Comment by maxtaco** [comment]
Happy to fix it. BTW, C/i will break, I need to fix some tests...

**Comment by keybase-ci-visdiff** [comment]
The commits 08c6830a0bef...d260c66139d1 introduce visual changes on linux. <details><summary>:mag_right: 2 changed</summary>

 * Changed: **Paper_Key-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1480523606236-08c6830a0bef-d260c66139d1-linux/Paper_Key-Normal-08c6830a0bef.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1480523606236-08c6830a0bef-d260c66139d1-linux/Paper_Key-Normal-d260c66139d1.png)  
   Paper_Key-Normal comparison
 * Changed: **Menubar-Private_Async_Writing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1480523606236-08c6830a0bef-d260c66139d1-linux/Menubar-Private_Async_Writing-08c6830a0bef.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1480523606236-08c6830a0bef-d260c66139d1-linux/Menubar-Private_Async_Writing-d260c66139d1.png)  
   Menubar-Private_Async_Writing comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 08c6830a0bef...90506727020f introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **Paper_Key-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1480528016184-08c6830a0bef-90506727020f-linux/Paper_Key-Normal-08c6830a0bef.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1480528016184-08c6830a0bef-90506727020f-linux/Paper_Key-Normal-90506727020f.png)  
   Paper_Key-Normal comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 08c6830a0bef...d8d1fa09b5b1 introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **Paper_Key-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1480530298891-08c6830a0bef-d8d1fa09b5b1-linux/Paper_Key-Normal-08c6830a0bef.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1480530298891-08c6830a0bef-d8d1fa09b5b1-linux/Paper_Key-Normal-d8d1fa09b5b1.png)  
   Paper_Key-Normal comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 08c6830a0bef...d8d1fa09b5b1 introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **Paper_Key-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1480532665381-08c6830a0bef-d8d1fa09b5b1-linux/Paper_Key-Normal-08c6830a0bef.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1480532665381-08c6830a0bef-d8d1fa09b5b1-linux/Paper_Key-Normal-d8d1fa09b5b1.png)  
   Paper_Key-Normal comparison
</summary>

**Pull request merged**